fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1135,6 +1138,9 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = ENABLE_FILTER_UNUSED_COLUMNS_IN_SCAN_STAGE)
     private boolean enableFilterUnusedColumnsInScanStage = true;
 
+    @VariableMgr.VarAttr(name = ENABLE_PRUNE_COLUMN_AFTER_INDEX_FILTER)
+    private boolean enablePruneColumnAfterIndexFilter = true;
```

Comment:
when should it be set to false?

Reply:
This is an emergency session variable to turn off the optimization if the optimization may has the bug.

Reply:
suggest change it to invisible

Reply:
Done, fixed it